pyvisa-py/gpib.py

```diff
@@ -68,16 +68,25 @@ def _inner(self):
 # TODO: Check board indices other than 0.
 BOARD = 0
```

Comment:
This is probably not needed anymore. Could you please remove it and any reference to it.